provider/aws/formation/resource/memcached.json.tmpl

```diff
@@ -29,7 +33,12 @@
     }
   },
   "Outputs": {
-    "Url": { "Value": { "Fn::Sub": "${CacheCluster.ConfigurationEndpoint.Address}:${CacheCluster.ConfigurationEndpoint.Port}" } }
+    "Url": { "Value": { "Fn::Sub": "${CacheCluster.ConfigurationEndpoint.Address}:${CacheCluster.ConfigurationEndpoint.Port}" } },
+    "User": { "Value": "" },
```

Comment:
If user/pass and name are always empty I thin we can just remove them

Reply:
If these were used then I'd say yes(like maybe Memcached would expose a different variable name for "User" and we convert the variable name to "User" to keep it consistent) but since it's never used I don't think we need it. Up to you though